packages/frontend/src/index.jsx

```diff
@@ -14,7 +14,7 @@ import { ToastProvider } from 'react-toast-notifications'
 import Group from './pages/group'
 import About from './pages/about'
 
-import GithubCorner from './components/github.html'
+//import GithubCorner from './components/github.html'
```

Comment:
Let's remove it completely
```suggestion
```

Reply:
Added execute permission to run the file!